src/TestUtils.jl

```diff
@@ -0,0 +1,74 @@
+module TestUtils
+
+using LinearAlgebra
+using Random
+using Test
+
+using Distributions
+using LogExpFunctions: logistic
+
+using AbstractGPs
+using ApproximateGPs
```

Comment:
Stupid style thing -- please put these in alphabetical order.

Reply:
They are; I just separated them by stdlib / external packages / our packages - you don't like that ? I thought it made it much clearer...

Reply:
Ahh fair enough. I honestly don't mind either way now that I see what you're doing, so I'm going to resolve this conversation 😂